### PR TITLE
Solved Responsive issues of Current-opening section(15 NOV)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -48,46 +48,28 @@
         <!-- join us section ends -->
 
         <!-- current opening section starts -->
-        <section>
-            <div class = "services-section montserrat-font-family overflow-hidden sky-light padding-75-0-0 xxs-padding-30">
```
Comment (amitjain24524) [inline careers.html]: rename section class name to current-openings-section
remove unwanted style or class name

File: careers.html [hunk 2]
```diff
@@ -48,46 +48,28 @@
...
-                <div class="container-fluid padding-0" id="current-opening-wrapper">
-                    <div id="opening-title-wrapper"></div>
-                    <div id="modal-content-wrapper"></div>
-                    <div class="mosaic jQueryMosaic" id="mosaic"></div>
-                        <div class = "services-section-head">
-                            <div id="services-title"></div>
```
Comment (amitjain24524) [inline careers.html]: rename section class name to current-openings-title

File: careers.html [hunk 3]
```diff
@@ -48,46 +48,28 @@
         <!-- join us section ends -->
 
         <!-- current opening section starts -->
-        <section>
-            <div class = "services-section montserrat-font-family overflow-hidden sky-light padding-75-0-0 xxs-padding-30">
-                <div class="container-fluid padding-0" id="current-opening-wrapper">
-                    <div id="opening-title-wrapper"></div>
-                    <div id="modal-content-wrapper"></div>
-                    <div class="mosaic jQueryMosaic" id="mosaic"></div>
-                        <div class = "services-section-head">
```
Comment (amitjain24524) [inline careers.html]: rename section class name to current-openings-heading

File: careers.html [hunk 4]
```diff
@@ -48,46 +48,28 @@
...
-                            <div id="services-title"></div>
-                            <section class="client-container text-center">
-                            </section>
-                        </div>
+            <section>
+                <div class="services-section montserrat-font-family sky-light xxs-padding-30">
+                    <div class="container-fluid padding-0" id="current-opening-wrapper">
+                        <div id="opening-title-wrapper"></div>
+                        <div id="modal-content-wrapper"></div>
+                        <div class="mosaic jQueryMosaic" id="mosaic"></div>
```
Comment (amitjain24524) [inline careers.html]: remove this line if a mosaic is not use.

File: css/demo.css [hunk 5]
```diff
@@ -3560,4 +3568,17 @@ svg{
...
+}
+.job-opening-image:last-child{
+    height: 400px;
+    width: 100%;
+    background-position: 70%;
+}
 
+/*#jobopening>div:last-child {*/
+/*    width: 100%;*/
+/*}*/
```
Comment (amitjain24524) [inline css/demo.css]: Remove commented code.

File: careers.html (hunk 1 above)
Comment (Palakagrawal28) [reply careers.html]: done

Comment (Palakagrawal28) [reply careers.html]: done

File: careers.html (hunk 2 above)
Comment (Palakagrawal28) [reply careers.html]: Removed this code(useless)

File: careers.html (hunk 4 above)
Comment (Palakagrawal28) [reply careers.html]: Removed

**Pull request merged**